AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/backgroundfilter/BackgroundFilterProcessor.swift

```diff
@@ -89,7 +93,7 @@ public class BackgroundFilterProcessor {
             logger.info(msg: "Updating buffer pool with new sizes: \(inputFrameCG.width) x \(inputFrameCG.height)")
             updateBufferPool(newWidth: inputFrameCG.width, newHeight: inputFrameCG.height)
             // Initialize the segmentationProcessor if it has not been initialized.
-            let initializeResult: Bool = segmentationProcessor.initialize(height: segmentationProcessorHeight,
+            let initializeResult: Bool = segmentationProcessor.initialize(withHeight: segmentationProcessorHeight,
```

Comment:
I feel technically this caused a breaking change due to the parameter name change. Is there any way we can avoid this or we should mark it breaking. Also don't forget to update CHANGELOG, and / or any document that should update with the new name.

Reply:
Good point. Let me just make it consistent with what it was before.